services/server/youtube.ts

```diff
@@ -1,12 +1,12 @@
-type PlayListType = {
+export type PlayListType = {
   title: string;
   link: string;
-}[];
+};
 
 export async function getYoutubePlaylist(
   playlistId: string,
   videosCount: number
-): Promise<PlayListType> {
+): Promise<PlayListType[]> {
```

Comment:
Does this endpoint return a list of playlists?
If it returns an array of videos then the typename should be changed

Reply:
Yes - I changed the name to PlaylistArray